dashboard/src/components/Filter/CheckboxSection.tsx

```diff
@@ -34,6 +36,12 @@ export interface ICheckboxSection {
   className?: string;
 }
 
+export interface Section {
```

Comment:
I didn't understand the need for this new interface, it seems to be an interface for a checkbox section, but we already have ICheckboxSection right above this. If this is correct, I would suggest changing the name of this interface to be clearer

Reply:
I create this interface to type a object that is used to construct a ICheckboxSection object. Like
`const checkboxSectionsProps: ICheckboxSection[] = useMemo(
    () =>
      sections.map(section => ({
        title: intl.formatMessage({ id: section.title }),
        subtitle: intl.formatMessage({ id: section.subtitle }),
        items: parsedFilter[section.sectionKey],
        onClickItem: (value: string): void => {
          setDiffFilter(old =>
            changeCheckboxFilterValue(old, section.sectionKey, value),
          );
        },
      })),
    [intl, parsedFilter, sections, setDiffFilter],
  );`

So, I will rename the interface to ISectionItem to be clearer